### PR TITLE
Update Gitlab CICD

### PR DESCRIPTION
**Pull request by fritz-astronomer**
Customer reports small modification was required to install astro cli in Gitlab CICD

### CODE REVIEW

**Review by kwcanuck — APPROVED** [review]

File: astro/ci-cd.md [hunk 1]
```diff
@@ -1019,13 +1026,13 @@ To automate code deploys to a Deployment using [GitLab](https://gitlab.com/), co
       services:
        - docker:dind
       variables:
-         ASTRONOMER_KEY_ID: $ASTRONOMER_KEY_ID
-         ASTRONOMER_KEY_SECRET: $ASTRONOMER_KEY_SECRET
+         ASTRONOMER_KEY_ID: ${ASTRONOMER_KEY_ID}
+         ASTRONOMER_KEY_SECRET: ${ASTRONOMER_KEY_SECRET}
```
Comment (jwitz) [inline astro/ci-cd.md:1030]:
```suggestion
         ASTRONOMER_KEY_ID: ${siteVariables.keyid}
         ASTRONOMER_KEY_SECRET: ${siteVariables.keyid}
```

File: astro/ci-cd.md [hunk 2]
```diff
@@ -1059,13 +1066,13 @@ When you create environment variables that will be used in multiple branches, yo
         services:
           - docker:dind
         variables:
-            ASTRONOMER_KEY_ID: $DEV_ASTRONOMER_KEY_ID
-            ASTRONOMER_KEY_SECRET: $DEV_ASTRONOMER_KEY_SECRET
+            ASTRONOMER_KEY_ID: ${DEV_ASTRONOMER_KEY_ID}
+            ASTRONOMER_KEY_SECRET: ${DEV_ASTRONOMER_KEY_SECRET}
```
Comment (jwitz) [inline astro/ci-cd.md:1070]:
```suggestion
            ASTRONOMER_KEY_ID: ${siteVariables.devkeyid}
            ASTRONOMER_KEY_SECRET: ${siteVariables.devkeysecret}
```

File: astro/ci-cd.md [hunk 3]
```diff
@@ -1076,13 +1083,13 @@ When you create environment variables that will be used in multiple branches, yo
         services:
           - docker:dind
         variables:
-            ASTRONOMER_KEY_ID: $PROD_ASTRONOMER_KEY_ID
-            ASTRONOMER_KEY_SECRET: $PROD_ASTRONOMER_KEY_SECRET
+            ASTRONOMER_KEY_ID: ${PROD_ASTRONOMER_KEY_ID}
+            ASTRONOMER_KEY_SECRET: ${PROD_ASTRONOMER_KEY_SECRET}
```
Comment (jwitz) [inline astro/ci-cd.md:1087]:
```suggestion
            ASTRONOMER_KEY_ID: ${siteVariables.prodkeyid}
            ASTRONOMER_KEY_SECRET: ${siteVariables.prodkeysecret}
```

File: astro/ci-cd.md [hunk 4]
```diff
@@ -994,6 +994,13 @@ This pipeline configuration requires:
 
 ### GitLab
 
+
+:::caution
+
+You may need certain tags or modifications based on your Gitlab Runner, if you are using hosted runners. Speak with your Gitlab Administrator if this is the case before starting, and review the [Gitlab Documentation](https://docs.gitlab.com/ee/ci/docker/using_docker_build.html) for building specifically in your use case
```
Comment (jwitz) [inline astro/ci-cd.md:1000]:
```suggestion
If you use hosted runners, you might need certain tags or modifications based on your Gitlab runner. Speak with your Gitlab administrator and review the [Gitlab Documentation](https://docs.gitlab.com/ee/ci/docker/using_docker_build.html) to customize this template for your use case.
```

**Review by jwitz — COMMENTED** [review]
Looks good @fritz-astronomer I just added some environment variables so the script would render correctly in Docusaurus. Planning to get rid of these environment variables now that we have a more stable install link for curl installs, but will hold on that until we merge these fixes!

File: astro/ci-cd.md [hunk 5]
```diff
@@ -994,6 +994,13 @@ This pipeline configuration requires:
 
 ### GitLab
 
+
+:::caution
```
Comment (jwitz) [inline astro/ci-cd.md:998]:
```suggestion
:::info
```

**Comment by fritz-astronomer** [comment]
@jwitz 👍 ?

**Pull request merged**